### ISSUE TITLE
Allow special aesthetic for relative positioning

### ISSUE THREAD

**Issue by OriolAbril** [arviz-devs/arviz-plots#95]
Not very clear on what we want this to be. I think a good starting point would be allowing a `yrelative` (or some better name) in `plot_dist` which is then converted into an actual y value multiplying by the maximium of the kde/hist/ecdf inside plot_dist.

Idea of steps involved:

1. User provides `yrelative` values
2. aesthetics are generated for `yrelative` which can be retrived as dataset
3. yrelative dataset is multiplied by density.max(some_dims) -> xarray handles all aligment and broadcasting
4. y aesthetic is created and yrelative deleted or yrelative gets its values overwritten and the aesthetic renamed (whatever we find easier to implement)
5. Plotting backends only see `y` aesthetic

This might not 100% work with the exact relative ratios we'd want but I think it will generally be good enough and probably much easier than attempting to handle relative spacing across backends. But it is also true I have literally no idea how to use relative units in bokeh nor plotly, so it is always possible that is easier than the steps above.

**Comment by aloctavodia** [comment]
I agree in general, but just providing `y` may be enough, as we then scale them by max density. At least for plots like plot_dist, it makes sense to always think of `y` in relative terms.

**Comment by menacingly-coded** [comment]
Yes we can normalize y and scale it using density.max(), eliminating the need for yrelative. But if yrelative is necessary, we can conduct tests and possibly introduce a backend compatibility layer.
we can also test if Bokeh and Plotly have built-in relative scaling support before implementing yrelative.

I would like to work on this issue. @aloctavodia please assign me this.

**Comment by aloctavodia** [comment]
Assigned. 

To make things more concrete, see this plot https://arviz-plots.readthedocs.io/en/latest/gallery/plot_psense.html. The goal is to position the point-intervals so they looks nice (approximately equally spaced) irrespective of the max-value of the kde (or other representations).

**Comment by menacingly-coded** [comment]
#157
